.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "**" ]
+
+name: Continuous Integration
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+          - nightly
+          - 1.40.0  # MSRV
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
I see that this action latest version is 2 (https://github.com/actions/checkout). Is there any specific reason why is it better to stick with v1?